source/Personality.cpp

```diff
@@ -148,11 +152,59 @@ void Personality::Load(const DataNode &node)
 		if(child.Token(0) == "confusion")
 		{
 			if(add || remove)
-				child.PrintTrace("Cannot \"" + node.Token(0) + "\" a confusion value:");
-			else if(child.Size() < 2)
-				child.PrintTrace("Skipping \"confusion\" tag with no value specified:");
-			else
+				child.PrintTrace("Cannot \"" + node.Token(0) + "\" a confusion node:");
+			// Accept the old method of defining confusion for backwards compatibility.
+			else if(child.Size() == 2)
 				confusionMultiplier = child.Value(1);
+			else
```

Comment:
I think it would be more straightforward to check both locations. That is, something like this should be valid:
```
confusion 10
	gain 180
```
Whereas right now you have it ignoring the children if a value is given.